src/types/AppliedReceipt2.ts

```diff
@@ -32,41 +32,41 @@ export function serializeAppliedReceipt2(
     stream.writeUInt16(TypeIdentifierEnum.cAppliedReceipt2)
   }
   stream.writeUInt8(cAppliedReceipt2Version)
-  // stream.writeString(obj.txid)
-  // stream.writeUInt8(obj.result ? 1 : 0)
-  // serializeAppliedVote(stream, obj.appliedVote)
-  // serializeConfirmOrChallengeMessage(stream, obj.confirmOrChallenge)
-  // stream.writeUInt16(obj.signatures.length)
-  // // TODO: Convert to for loop
-  // obj.signatures.forEach((sig) => serializeSign(stream, sig))
-  // stream.writeString(obj.app_data_hash)
-  const stringified = Utils.safeStringify(obj)
-  stream.writeString(stringified)
+  stream.writeString(obj.txid)
+  stream.writeUInt8(obj.result ? 1 : 0)
+  serializeAppliedVote(stream, obj.appliedVote)
+  serializeConfirmOrChallengeMessage(stream, obj.confirmOrChallenge)
```

Comment:
AFAIK it is not used in POQo

Reply:
I am pulling in some more commits will see if that helps this

Reply:
found out this was missed.  have a task to solve the issue

Reply:
will be in a separate pr